optaplanner-core/src/main/java/org/optaplanner/core/impl/constructionheuristic/DefaultConstructionHeuristicPhase.java

```diff
@@ -36,7 +36,7 @@
         implements ConstructionHeuristicPhase<Solution_> {
 
     protected EntityPlacer entityPlacer;
-    protected ConstructionHeuristicDecider decider;
+    protected ConstructionHeuristicDecider<Solution_> decider;
```

Comment:
Was this necessary or does it simply improve the code? I +1 this but I wonder why not to improve type safety in other related classes like ConstructionHeuristicForager, LocalSearchForager etc.

Reply:
+1, but maybe it deserves a separate PR

Reply:
This was to make it consistent with the LocalSearchDecider usage.